tracer/src/Datadog.Trace/AppSec/Coordinator/SecurityCoordinatorHelpers.Core.cs

```diff
@@ -123,17 +127,16 @@ internal static void CheckPathParamsFromAction(this Security security, HttpConte
         }
     }
 
-    internal static object? CheckBody(this Security security, HttpContext context, Span span, object body)
+    internal static object? CheckBody(this Security security, HttpContext context, Span span, object body, bool response = false)
```

Comment:
I would strongly suggest not making this a default parameter. It's a recipe for forgetting to pass it when it should be otherwise :D